lambdas/preview/index.py

```diff
@@ -173,6 +173,7 @@ def extract_csv(head, separator):
     except pandas.errors.ParserError:
         data = pandas.read_csv(
             io.StringIO('\n'.join(head)),
+            error_bad_lines=False,
```

Comment:
This will cause the preview to silently not accurately match the underlying data. This will be misleading and users will consider it to be a bug. 

I don't think this is the right solution

Reply:
I can either accumulate the warnings + errors, and send those to the HTML response, or we can run with `quoting=3` in the second pass. Let me investigate.

Reply:
Fixed; I'll file for the catalog to pick the errors up